lib/js/admin-geo.js

```diff
@@ -91,11 +102,17 @@
         marker.setPosition(loc);
         $('#location_lat').val(latitude.toString());
         $('#location_lng').val(longitude.toString());
+        $('#location_lat').trigger("focus");
+        $('#location_lat').trigger("change");
+        $('#location_lat').trigger("blur");
+        $('#location_lng').trigger("focus");
+        $('#location_lng').trigger("change");
+        $('#location_lng').trigger("blur");
```

Comment:
@loetie Is this necessary?

Reply:
It was needed so the default config save action was triggered when the hidden fields got filled with a new value. So its to simulate that a field has changed and left.